### PR TITLE
Restructuring

### PR DESCRIPTION
**Pull request by jistr**
The main thing in this PR is splitting out Keystone adoption in a separate document. As is apparent from the Glance adoption doc, there may be enough configuration info for each service that we should get back to the document-per-service model. Even if eventually we may be able to patch the OpenStackControlPlane CR for all services at once, the documentation structure will be much clearer if we address each service in its own doc, rather than having one mega-document.

Additionally this PR tweaks MariaDB prerequisite info, puts setting of all service passwords into a common place right after we create the `osp-secret` at the very beginning of the procedure, and tweaks the Glance doc mainly in headings to be more consistent with the rest of the documentation.

### CODE REVIEW

**Comment by jistr** [comment]
This is probably easier to review by looking at the branch content rather than the git diff: https://github.com/jistr/data-plane-adoption/tree/f/restructure

File: README.md
Comment (fmount) [inline README.md:15]: Before `MariaDB` copy, we can have a prereq section where we assume TripleO is up && running on one side, and the basic ctlplane is running on the other side: the first thing to do is run crc with all the `disabled` services.
iirc this last topic is covered by [Deploy OpenStack control plane services](https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/backend_services_deployment.md), and it should go first, right?

**Review by fao89 — APPROVED** [review]

**Comment by fao89** [comment]
Do we want to publish this doc?
if so, I can set mkdocs like this:
https://openstack-k8s-operators.github.io/openstack-ansibleee-operator/

**Comment by fmount** [comment]
> Do we want to publish this doc? if so, I can set mkdocs like this: https://openstack-k8s-operators.github.io/openstack-ansibleee-operator/

It's not a bad idea, but I'll let @jistr comment on this! Long term, I like the idea of mkdocs!

File: README.md
Comment (jistr) [reply README.md:15]: That's pretty much all in "Deploy podified backend services" which is already the step right before MariaDB copy. I could expand the prereq list to mention that we need to have the cloud we want to Adopt up and running.

**Comment by jistr** [comment]
> Do we want to publish this doc? if so, I can set mkdocs like this: https://openstack-k8s-operators.github.io/openstack-ansibleee-operator/

I'd like to explore & chat about what it would mean re structuring of the content, ease of contribution, and how to generate the site locally, before we jump onto that. If we use a static site generator, there should be an easy way to generate the site locally, perhaps utilizing a Makefile + virtualenv. I'd like to explore if we get enough benefits over the built-in Github markdown renderer without causing much hassle for contributors/maintainers. Also i'm curious what other projects in the org (beyond Ansible EE Operator) plan to do regarding their docs. It would be good to get consistency. I like the sidebar menu there. So i'm leaning towards +1 but let's discuss.

**Comment by fmount** [comment]
> > Do we want to publish this doc? if so, I can set mkdocs like this: https://openstack-k8s-operators.github.io/openstack-ansibleee-operator/
> 
> I'd like to explore & chat about what it would mean re structuring of the content, ease of contribution, and how to generate the site locally, before we jump onto that. If we use a static site generator, there should be an easy way to generate the site locally, perhaps utilizing a Makefile + virtualenv. I'd like to explore if we get enough benefits over the built-in Github markdown renderer without causing much hassle for contributors/maintainers. Also i'm curious what other projects in the org (beyond Ansible EE Operator) plan to do regarding their docs. It would be good to get consistency. I like the sidebar menu there. So i'm leaning towards +1 but let's discuss.

Yeah let's discuss and follow up, but definitely the restructuring patch looks good!

File: README.md
Comment (fmount) [reply README.md:15]: Thanks for clarifying! LGTM.

**Review by fmount — APPROVED** [review]

File: README.md
Comment (jistr) [reply README.md:15]: I'll add one more commit mentioning that the cloud which we want to adopt should be up and running, and it has to be on Wallaby. And i'll merge.

**Comment by jistr** [comment]
Thanks for the reviews! :)

**Pull request merged**
